e2e/config/services_shannon.yaml

```diff
@@ -478,6 +478,14 @@ services:
       transaction_hash: "0xc65cff6db22a350d39d9154be650cd10bd0ca1c143402c51b3c7802f702328b0"
       call_data: "0x18160ddd"
 
+  # ------------- Anvil Services -------------
```

Comment:
Do you mind adding a new definition for `anvil` to the `definitions` list in:
https://github.com/buildwithgrove/path/blob/feat-load-test-anvil/e2e/config/services.schema.yaml#L20